metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -294,11 +294,27 @@
   (get-granule-concept-ids
    [db provider native-id]
    (let [table (tables/get-table-name provider :granule)
-         result (su/find-one
-                 db (select [:concept-id :parent-collection-id]
-                            (from table)
-                            (where (by-provider :granule provider `(= :native-id ~native-id)))))]
-     [(:concept_id result) (:parent_collection_id result)]))
+         {:keys [provider-id small]} provider
+         stmt (if small
```

Comment:
Consider breaking this `if` statement out into its own function. Something like `stmt (prepare-granule-concept-id-query <args>)` would feel much easier to read to me

Reply:
I intentionally keep the sql statements together, so we can see what is different between the regular provider query and the small provider query. It is done the same way in other functions too in the same namespace. I think we should keep it this way.

Reply:
That's fair, I just like to try keeping actual logic out of `let` blocks. Functions feel much easier to reason about if there are more, simpler functions.